src/index.ts

```diff
@@ -1215,19 +1216,19 @@ export interface SeatingChart {
     changeConfig: (config: ConfigChange) => Promise<void>
     clearSelection: () => Promise<void>
     deselectCategories: (categoryIds: string[]) => Promise<void>
-    deselectObjects: (objects: string[] | Selection[]) => Promise<void>
+    deselectObjects: (objects: (string | Selection)[]) => Promise<void>
     destroy: () => void
     findObject: (label: string) => Promise<SelectableObjectProps>
     getReportBySelectability: () => Promise<Object>
     holdToken: string
     listCategories: () => Promise<Category[]>
-    listSelectedObjects: () => Promise<(any)[]>
+    listSelectedObjects: <T extends SelectableObjectProps>() => Promise<T[]>
```

Comment:
Is there any difference with just `() => Promise<SelectableObjectProps[]>`? Don't know TS well enough 😉

Reply:
Yes, any time you use `any`, you basically tell TypeScript to forego any type checking (and thus you don't get any suggestions on the returned objects).

Reply:
Yes, but I mean: what the difference between using that generic `T` parameter, and writing `() => Promise<SelectableObjectProps[]>`? Or is there's an `any` involved then?

Reply:
Oh I see. No, it just indicates that the type can be anything that adheres to the extended interface - functionally it would not make a difference here since this called at runtime. Depends what we want to do, give a hint that this can be different type of objects, or keep it simpler. We'd need some type predicates to narrow down the types anyway.

Reply:
Yeah, maybe it's good to emphasise that any subtype of `SelectableObjectProps` can get returned 👍 Let's do the same for `findObject` then though.